protzilla/data_integration/di_plots.py

```diff
@@ -7,7 +7,7 @@
 from protzilla.constants.protzilla_logging import logger
 from protzilla.utilities.utilities import fig_to_base64
 
-from ..constants.colors import PROTZILLA_DISCRETE_COLOR_SEQUENCE
+from ..constants.colors import *
```

Comment:
hier auch die gleiche Bemerkung wie oben, bzw könnte man hier halt genau nur die eine Konstante importieren, da nur die verwendet wird und in anderen Files wo mehr verwendet wird, dann alle importieren